docs/GettingStarted/Examples.md

```diff
@@ -196,3 +196,21 @@ spec:
           - name: JIRA_RESOLVED_STATUS
             value: DONE
 ```
+
+## Webhook exporter only
+
+In this example, Pelorus will receive data from the external systems without making any API calls to OCP nor external providers.
```

Comment:
OpenShift instead of OCP? (I get confused with these abbreviations)

Reply:
changed.